public/app/features/dashboard/state/DashboardModel.ts

```diff
@@ -114,6 +115,9 @@ export class DashboardModel {
     panelInView: true,
     getVariablesFromState: true,
     formatDate: true,
+    hasChangesThatAffectsAllPanels: true,
+    changeAffectsAllPanels: true,
```

Comment:
you do not need to add functions here, only value props

Reply:
Gotcha, thank you!